.github/workflows/ci.yml

```diff
@@ -5,6 +5,21 @@ name: CI
 on: [push]
 
 jobs:
+  format-cmake:
```

Comment:
Do you know if it would be possible to make this for `on: pull_request`? In my personal experience (rust projects mostly and WebKit style bot) the CI on badly formatted PRs should be a red flag, so I'd rather run this job on PRs rather than on `push` scope, unless I misunderstood what `push` means :)

Reply:
Nice, I see the new check here, cool stuff Don, thanks!